src/Twilio/Http/CurlClient.php

```diff
@@ -9,26 +9,44 @@
 class CurlClient implements Client {
     public const DEFAULT_TIMEOUT = 60;
     protected $curlOptions = [];
-    protected $debugHttp = false;
 
     public $lastRequest;
     public $lastResponse;
 
     public function __construct(array $options = []) {
         $this->curlOptions = $options;
-        $this->debugHttp = \getenv('DEBUG_HTTP_TRAFFIC') === 'true';
     }
 
     public function request(string $method, string $url,
                             array $params = [], array $data = [], array $headers = [],
                             string $user = null, string $password = null,
-                            int $timeout = null): Response {
+                            int $timeout = null, string $logLevel = null): Response {
         $options = $this->options($method, $url, $params, $data, $headers,
                                   $user, $password, $timeout);
 
         $this->lastRequest = $options;
         $this->lastResponse = null;
 
+        if (getenv('DEBUG_HTTP_TRAFFIC') === 'true') {
```

Comment:
Should we use `get/setLogLevel` here and below (L34)? If so, this logic could go in `getLogLevel`.

Reply:
Based on my understanding don't have access the `Twilio/Rest/Client` setters/getters from this http client which is why I passed in the `logLevel` when the request is being created in the rest client.